### ISSUE TITLE
Dev "Load from disk" path incorrect in institution admin

### ISSUE THREAD

**Issue by trombonekenny** [PrairieLearn/PrairieLearn#10379]
I happened to be on the institution admin page (`/pl/administrator/institution/1`) in local dev when I needed to `Load from disk`. I got a 404 because the button there tried to load `/pl/administrator/institution/1/loadFromDisk`  Is some variable not getting set?

**Comment by jonatanschroeder** [comment]
It seems the issue is that "Load from disk" assumes there is a `${urlPrefix}/loadFromDisk` for any value of `urlPrefix`, which is not the case in the institution administrator page. I think the easiest solution here is to add the path for that route, or change the link to always use the `plainUrlPrefix` version. Is there a reason why `loadFromDisk` should have multiple routes?

**Comment by nwalters512** [comment]
The only reason I can think of is that it'll show the appropriate navbar for the url prefix. I don't think this is a particularly compelling reason and I think we'd be fine with a single mount at `planUrlPrefix`. cc @mwest1066 I think this aligns with how we're thinking about course navigation in the future (e.g. no more double-mounting of routes if we can avoid it), does that sound right?

**Comment by mwest1066** [comment]
I agree that using `plainUrlPrefix` sounds like the right fix here. I'd love to get away from double-mounting (perhaps as part of #10608) but I'm not currently sure how to do that while retaining course instance context.
